### PR TITLE
Renames Robust Nukie

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
The most dumb, uncreative name you could probably give this drink. If you manifested the collective OOC brainrot of lizard into the name of a drink then this would be it. PRO SOSA NUKIES ROBUST ROBUST NUKIES PRO SOSA TOOLBOXING ROBUST SHITSEC

**Media**
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by DogZeroX** [comment]
Flareguy don't start your PR description with "What it says on the tip" challenge (impossible)

**Comment by deltanedas** [comment]
brewn is not a word :trollface:

**Comment by Equivocateur** [comment]
Brewed is

**Comment by Vaaankas** [comment]
get robust

**Comment by Jackal298** [comment]
Leave the old name, as for me it is more suitable for a drink in a can

**Comment by Emisse** [comment]
i like the rename but i like the rant in the description about this more

**Comment by potato1234x** [comment]
the sprite of the can needs more attention than the name tbh

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_cans.yml [hunk 1]
```diff
@@ -298,8 +298,8 @@
 - type: entity
   parent: DrinkCanBaseFull
   id: DrinkNukieCan
-  name: robust nukie drink
-  description: An encouraging drink... very encouraging. It is filled with green liquid, after use it is recommended to consult a doctor.
+  name: blood-red brew can
+  description: A home-brewed drink made from the crazed minds at the Syndicate. Not reccomended by doctors.
```
Comment (MisterMecky) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_cans.yml:302]:
```suggestion
  description: A home-brewed drink made from the crazed minds at the Syndicate. Not recommended by doctors.
```

**Pull request merged**